### PR TITLE
Fix error on non existing thumbnail deletion

### PR DESCRIPTION
**Pull request by aloupfor**

### CODE REVIEW

File: MediaAdmin/EventSubscriber/MediaDeletedSubscriber.php [hunk 1]
```diff
@@ -41,7 +41,10 @@ public function deleteAlternatives()
     {
         /** @var MediaInterface $media */
         foreach ($this->medias as $media) {
-            $this->fileAlternativesManager->deleteThumbnail($media);
+            if (null !== $media->getThumbnail()) {
```
Comment (itkg-alavieille) [inline MediaAdmin/EventSubscriber/MediaDeletedSubscriber.php]: I think, It is not a good idea whatsoever the manager who decides to launch the strategies or not. for instance a developer wants to be a strategy that puts a default image in the case where there aren't thumbnail

Comment (nicolasThal) [inline MediaAdmin/EventSubscriber/MediaDeletedSubscriber.php]: I agree with @itkg-alavieille , each strategy should do the check if they want to. Neither the subscriber nor the strategy manager should make such a test

**Comment by aloupfor** [comment]
@nicolasThal @itkg-alavieille

**Comment by alavieille** [comment]
can you add this case in the test ?

**Comment by aloupfor** [comment]
@itkg-alavieille

**Comment by alavieille** [comment]
:+1:

**Pull request merged**